js/models/Profile.js

```diff
@@ -6,6 +6,10 @@ import SocialAccounts from '../collections/SocialAccounts';
 export default class extends BaseModel {
   defaults() {
     return {
+      name: `ob ${Math.random().toString(36).slice(2)}`,
```

Comment:
Ok, sounds good. There is also an underscore unique id generator you might be interested in:
http://underscorejs.org/#uniqueId

What's here is probably fine, so I'm just sharing with you mostly as an FYI.